### ISSUE TITLE
Remove Python-specific reference terms from reference import

### ISSUE THREAD

**Issue by jeremydouglass** [gaocegege/Processing.R#120]
Currently the reference is seeded by importing reference entries, description text, and cross-references from the Python mode.

However, Python mode contains many Python-specific reference entries which are not in the Processing(Java) API, and which will not be in Python.R. For example:

-  dict_items
-  dict_iteritems
-  dict_keys
-  dict_update
-  dict_values
-  dictionary

https://github.com/gaocegege/Processing.R/commit/70009bc688bb8bd84be2496e67009ac737a3a510#diff-67374dee80d03cf169846122755bef45

Some python-specific entries include:

-  Composite `tuple`, `dictionary`, `string`
-  Dictionary Methods
-  List Methods
-  List Functions
-  String Methods

https://processing-r.github.io/Processing.R-docs/

These need to be removed -- either through an ignore list for the automatic import script, or, if the import was one-time-only, then manually removed (along with any cross-references).

Note however that if there ever is a re-import then we have to clean all over again, so an ignore list (processed by either the import script or a cleanup removal script) might be better approach than a manual cleanup commit.

**Comment by gaocegege** [comment]
Yeah, and there are some items about `float` and built-in types, which we also need to remove.

**Comment by jeremydouglass** [comment]
Good work, but I think we need to do a bit more work on this before closing the issue, and there should be a pull request review.

Python-specific reference terms are keywords from the Python language.

For example, `def` -- this is a Python keyword:

-  https://github.com/gaocegege/Processing.R/blob/master/examples/reference/def/.property.yml

Another example, `add_library` -- this is specific to Python mode, until we decided to implement something like it in R (which might not have the same name).

-  https://github.com/gaocegege/Processing.R/blob/master/examples/reference/add_library/.property.yml

Another example, `False` -- Processing(Java) uses built-in `boolean` `false` `true`; Python uses `Boolean`, `False`, `True`; I *believe* that R uses the built-ins `FALSE` and `TRUE`.

-  https://github.com/gaocegege/Processing.R/blob/master/examples/reference/False/.property.yml

While reviewing this issue, see also my comments on recent pull requests:

-  https://github.com/gaocegege/Processing.R/pull/130
-  https://github.com/gaocegege/Processing.R/pull/132

**Comment by gaocegege** [comment]
Sorry, my mistake.

**Comment by gaocegege** [comment]
* https://processing.org/reference/nf_.html
* https://processing.org/reference/nfc_.html
* https://processing.org/reference/nfp_.html
* https://processing.org/reference/nfs_.html

**Comment by jeremydouglass** [comment]
Currently I believe that all Python references have been removed from the docs, and R references (including importLibrary) were added. Can this issue be closed?

**Comment by jeremydouglass** [comment]
Actually, I see `importLibrary` in the reference, but not `library`. Perhaps open adding that as a separate issue.

**Comment by gaocegege** [comment]
No, we miss the functions `nf` and so on.

**Comment by gaocegege** [comment]
library is not a valid item in our mode, IMO. :thinking:

**Comment by jeremydouglass** [comment]
> No, we miss the function nf and so on.

Got it. That will be good for a community contribution.

> library is not a valid item in our mode, IMO.

I don't understand what you mean. We have a whole section on the repo homepage about `library`:

> ### R Packages: library()
> 
> Processing.R has limited support for R packages. It will automatically download R packages that are requested using the library() function, so you can use packages directly.

...but we don't have `library` listed on the reference page.

**Comment by gaocegege** [comment]
Oh, my mistake. Yeah we don't have the doc for `library`
